packages/hagrid/hagrid/cli.py

```diff
@@ -2499,13 +2500,30 @@ def version() -> None:
     is_flag=True,
     help="CI Test Mode, don't hang on Jupyter",
 )
+@click.option(
+    "--repo",
+    default="openmined/pysyft",
```

Comment:
maybe we can use the `arg_cache` class to set the default repo and branch.
```
from .cache import arg_cache
# arg_cache.repo ->  for default branch
# arg_cache.branch -> for default branch
```

Reply:
We want the quickstart to workaround with the "dev" branch right now but the arg cache has the stable branch hence I'm keeping the branch as it is but using the default repo from here. Let me know what do you think of this?

Later, a small PR can be made to add the dev branch to `arg_defaults`

Reply:
For the repo, please take a look at this: https://github.com/OpenMined/PySyft/pull/6686/commits/3985efb83e863cec6f425e5e08fcdf43113d2700 and resolve if fine

Reply:
We can use `DEFAULT_BRANCH` (which points to dev)
from .cache import DEFAULT_BRANCH

Reply:
Thanks for this! Done via https://github.com/OpenMined/PySyft/pull/6686/commits/959a9b5f642c5eb12ab95a9151489d62c3c7c174